src/factory.ts

```diff
@@ -256,13 +259,19 @@ function createModelApi<
 
       return records.map(removeInternalProperties)
     },
-    toHandlers(type, baseUrl): any {
+    toHandlers(type: 'rest' | 'graphql', baseUrl: string): any {
       if (type === 'graphql') {
         return generateGraphQLHandlers(modelName, definition, api, baseUrl)
       }
 
       return generateRestHandlers(modelName, definition, api, baseUrl)
     },
+    toSchema(type: 'graphql') {
```

Comment:
Sure! I was assuming this would be the "least breaking" upgrade path in case you introduce stuff like OpenAPI schemas in the future, but I see why it could be confusing for users that discover the API without TypeScript.
I'll change it.

Reply:
Renamed it as requested :)